eZ/Bundle/EzPublishRestBundle/Tests/DependencyInjection/Compiler/OutputVisitorPassTest.php

```diff
@@ -53,4 +53,54 @@ public function testProcess()
         self::assertEquals('ezpublish_rest.output.visitor.test_string', $dispatcherMethodCalls[0][1][1]->__toString());
         self::assertEquals('ezpublish_rest.output.visitor.test_array', $dispatcherMethodCalls[1][1][1]->__toString());
     }
+
+    public function testPriority()
+    {
+        $regexp1 = array('(^.*/.*$)');
```

Comment:
The test would probably be more readable if you used one array for test data, with regexp as first element, and priority as second, optional one. You could then iterate over them to create the definitions instead of doing so sequentially.